src/compile/common.ts

```diff
@@ -176,3 +171,18 @@ export function titleMerger(v1: Explicit<string>, v2: Explicit<string>) {
       v1.value + ', ' + v2.value // join title with comma if different
   };
 }
+
+/**
+ * Checks whether a fieldDef for a particular channel requires a computed bin range.
+ */
+export function binRequiresRange(fieldDef: FieldDef<string>, channel: Channel) {
+  if (!fieldDef.bin) {
+    console.warn('Only use this method with binned field defs');
+    return false;
+  }
+  if (!isScaleChannel(channel)) {
+    // range is only needed for scales that have legends or axes
```

Comment:
This comment is pretty useless and confusing (as it doesn't covered the whole criteria).  I just push a fix to simplify it.

Reply:
Thanks